design/src/SUMMARY.md

```diff
@@ -66,7 +66,7 @@
   - [RFC-0037: The HTTP Wrapper](./rfcs/rfc0037_http_wrapper.md)
   - [RFC-0038: User-configurable retry classification](./rfcs/rfc0038_retry_classifier_customization.md)
   - [RFC-0039: Forward Compatible Errors](./rfcs/rfc0039_forward_compatible_errors.md)
-  - [RFC-0040: Behavior Major Versions](./rfcs/rfc0040_behavior_major_versions.md)
+  - [RFC-0040: Behavior Major Versions](./rfcs/rfc0040_behavior_versions.md)
```

Comment:
I assume you deliberately left `RFC-0040: Behavior Major Versions` here as opposed to `RFC-0040: Behavior Versions` for a reason.

Reply:
Oops, I need to fix this one.